### PR TITLE
Updates for MAS 9.0.x release

### PR DESCRIPTION
**Pull request by NatarajBTI**
Updates for MAS 9.0.x release

### CODE REVIEW

**Comment by NatarajBTI** [comment]
By mistake, during hung state of my laptop, mouse cursor has clicked on close button of this PR while navigating between tabs..reopening it..

**Comment by NatarajBTI** [comment]
@ocofaigh - Please review this PR..thanks

**Comment by ocofaigh** [comment]
@NatarajBTI Code looks good. Have you tried an upgrade test yet? I suggest before merging this we import your branch into your catalog so we can due an upgrade. I can help if you ping me in the morning. We need access to the account that your catalog is in

**Comment by NatarajBTI** [comment]
@ocofaigh - Thanks for your review comments.
Yes, I have already executed both update and upgrade scenarios from MAS DA 8.11.x to MAS 9.0.x using MAS CLI.
https://jsw.ibm.com/browse/MASHYPER-147

Here are the screenshots & log files if you don't have access to this JIRA project:

image

image

[Update_and_Upgrade_Logs_Screenshots.zip](https://github.com/user-attachments/files/17142352/Update_and_Upgrade_Logs_Screenshots.zip)

Please check this & let me know if we still have to run an upgrade scenario and surely can do it..thanks

File: ibm_catalog.json [hunk 1]
```diff
@@ -62,7 +62,7 @@
               ],
               "id": "95fccffc-ae3b-42df-b6d9-80be5914d852-global",
               "name": "deploy-arch-ibm-slz-ocp",
-              "version": ">=v3.0.0",
+              "version": ">=v3.0.0,<6.0.0",
```
Comment (ocofaigh) [inline ibm_catalog.json:65]: @NatarajBTI have you tried with v6.0.1 or later of landing zone? It should fix the issue you saw with storage. That means you can remove this upper constraint.

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**